packages/extension-api/src/extension-api.d.ts

```diff
@@ -3763,6 +3763,19 @@ declare module '@podman-desktop/api' {
 
   export interface NetworkCreateOptions {
     Name: string;
+    CheckDuplicate?: boolean;
```

Comment:
reading https://docs.podman.io/en/latest/_static/api.html?version=latest#tag/networks-(compat)/operation/NetworkCreate

this is deprecated, let's remove this


```suggestion
```